apm-sniffer/apm-sdk-plugin/solrj-7.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/solrj/commons/SolrjTags.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.skywalking.apm.plugin.solrj.commons;
+
+import org.apache.skywalking.apm.agent.core.context.tag.StringTag;
+import org.apache.skywalking.apm.agent.core.context.trace.AbstractSpan;
+
+public class SolrjTags {
+
+    //    public static StringTag DB_TYPE = new StringTag(1, "db.type");
+    public static StringTag TAG_QT = new StringTag(1,"qt");
+    public static StringTag TAG_PATH = new StringTag(2, "path");
+    public static StringTag TAG_COLLECTION = new StringTag(3, "collection");
+    public static StringTag TAG_ACTION = new StringTag(4, "action");
+    public static StringTag TAG_METHOD = new StringTag(5, "method");
+    public static StringTag TAG_COMMIT_WITHIN = new StringTag(6, "commitWithin");
+    public static StringTag TAG_STATUS = new StringTag(7, "status");
+    public static StringTag TAG_Q_TIME = new StringTag(8, "QTime");
+    public static StringTag TAG_NUM_FOUND = new StringTag(9, "numFound");
+
+
+
+    public static StringTag TAG_STATUS_CODE = new StringTag(10, "statusCode");
+    public static StringTag TAG_ELAPSE_TIME = new StringTag(11, "elapseTime");
+
+    public static StringTag TAG_CONTENT_TYPE = new StringTag(12, "contentType");
+    public static StringTag TAG_CONTENT_LENGTH = new StringTag(13, "contentLength");
+    public static StringTag TAG_CONTENT_ENCODING = new StringTag(14, "contentEncoding");
+
+    public static StringTag TAG_COMMIT = new StringTag(15, "commit");
+    public static StringTag TAG_SOFT_COMMIT = new StringTag(16, "softCommit");
+
+    public static StringTag TAG_OPTIMIZE = new StringTag(17, "optimize");
+    public static StringTag TAG_MAX_OPTIMIZE_SEGMENTS = new StringTag(18, "maxOptimizeSegs");
+
+    public static StringTag TAG_DELETE_TYPE = new StringTag(19, "delete.type");
+    public static StringTag TAG_DELETE_VALUE = new StringTag(20, "delete.by");
+    public static StringTag TAG_DOCS_SIZE = new StringTag(21, "docsSize");
```

Comment:
I will review the tag list later. Too many tags will cost too many cpu, network bandwidth, storage. 

We need to discuss about which ones are meaningful. In existing plugins, one plugin only has particular special tags, most are in common lib already.

Reply:
I have a talk about Tag with @wu-sheng off-line. The main idea is 'Too many tags will cost too many cpu, network bandwidth, storage.' 
So he gave the key idea how to reduce the number of tags. Depending on whether it is usefully to help finds out the bottleneck. 'Tracer is not logger.'

He clarifies the usage of tagId. It is usually used in the mutable value in the span. Likes HttpStatus in RPC.